### PR TITLE
Allow the use of --accountAlias in eval command

### PR DESCRIPTION
**Pull request by LeeBrotherston**
Makes use of account alias when using the eval command, so that 

`aws-sso eval --account <bunch o numbers> --role AdminFun`

can also be expressed in the form

`aws-sso eval --accountAlias production --role AdminFun`

### CODE REVIEW

**Comment by synfinatic** [comment]
Hey @LeeBrotherston,

So a few comments:

1. Have you looked into using `--profile` and the `ProfileFormat` config option?  Then you could just specify `aws-sso eval --profile production:AdminFun`.  Specifically `ProfileFormat: '{{ FirstItem .AccountName (.AccountAlias | nospace) }}:{{ .RoleName }}'`.   (This is mentioned in the [FAQ](https://synfinatic.github.io/aws-sso-cli/FAQ/#how-to-configure-profileformat) and will be the default value in aws-sso v2.0)
2. If you do that, then the bash/zsh shell script integration for [aws-sso-profile](https://synfinatic.github.io/aws-sso-cli/commands/#aws-sso-profile) works

Basically, I don't see the need to add another CLI argument when the existing `--profile` argument will effectively do what you want?

**Comment by LeeBrotherston** [comment]
Aha didn't realize that `ProfileFormat` was configurable and was looking for a cli switch.  Got it 👍🏻
